app/mailers/user_mailer.rb

```diff
@@ -1,10 +1,9 @@
-# rubocop:disable Metrics/ClassLength
 class UserMailer < ActionMailer::Base
   include Mailable
   include LocaleHelper
   before_action :attach_images
-  default from: email_with_name(Figaro.env.email_from, Figaro.env.email_from),
-          reply_to: email_with_name(Figaro.env.email_from, Figaro.env.email_from)
+  default from: email_with_name(Figaro.env.email_from, Figaro.env.email_from_display_name),
```

Comment:
This deploy contains the change to the email display name -- we should try to validate this on as many emails as possible to make sure we don't affect our deliverability